docs/architecture.md

```diff
@@ -0,0 +1,24 @@
+WISP consists of a set of loosly coupled, collaborating services. This structure is what's known as a `microservice` architecture where each service can be developed, tested, deployed, scaled, and maintained independently from the others.
```

Comment:
Add a bunch of links to this file. For example, when you first mention microservices, link to an explanation on microservices. More obviously: when you mention technologies like Express or Vue, etc., link to their homepages.

Atlas, NGINX, and others are more examples